src/components/SidePanel/NavGroup/FilterNavGroup/FilterGroup/index.tsx

```diff
@@ -91,14 +136,12 @@ const FilterGroup: React.FC<FilterGroupProps> = ({
               Cancel
             </Button>
             <Button
-              onClick={() => {
-                editFilter("name", newFilterName, filter);
-                resetEditState();
-              }}
+              disabled={!isValid}
```

Comment:
Can we also disable the button when the input is empty to prevent users from submitting an empty string `""` as a filter?